ttnn/ttnn/operations/unary.py

```diff
@@ -241,6 +239,15 @@ def _golden_function_elu(input_tensor_a, *args, alpha=1.0, **kwargs):
 ttnn.attach_golden_function(ttnn.elu, golden_function=_golden_function_elu)
 
 
+def _golden_function_hardtanh(input_tensor_a, *args, min_val=-1.0, max_val=1.0, **kwargs):
+    import torch
+
+    return torch.nn.functional.hardtanh(input_tensor_a, min_val=min_val, max_val=max_val)
```

Comment:
can it be like this instead?
```suggestion
def _golden_function_hardtanh(input_tensor_a, *args, **kwargs):
    import torch

    return torch.nn.functional.hardtanh(input_tensor_a, kwargs)
```

Reply:
sure! yes it can. I was just being thorough and explicit about the kwargs matching with torch api.